.clitest/functions/.test.teardown

```diff
@@ -0,0 +1,10 @@
+#!/bin/zsh
+
+# restore git
+(( $+functions[git] )) && unfunction git
```

Comment:
When using `&&` as an `if`, I prefer to add a line break after the `&&` plus indent the next line with 4 spaces. This makes your intention clearer and makes the diff easier to follow for Git, if you later decide to turn it into an actual `if`.